ts/components/Pinpad/KeyPad.tsx

```diff
@@ -118,7 +118,7 @@ const renderPinRow = (
           renderPinCol(
             el.e1,
             el.e2,
-            el.e1.length === 1 ? "digit" : "label",
+            RegExp(/^[0-9]$/).test(el.e1) ? "digit" : "label",
```

Comment:
@bfabio thank you again for your support ;)
However this is how it looks like on iOS if we apply the regex check
<img src="https://user-images.githubusercontent.com/822471/80965946-c2af1b80-8e13-11ea-8045-cead170604b3.png" height="600"/>

For digits we use a specific style so you could consider to keep the previous behavior based on length check. In that way we could continue to render the **DEL** button with a right font and the background too
This is what you will get
<img src="https://user-images.githubusercontent.com/822471/80966350-a95a9f00-8e14-11ea-9ce5-c8f7754f452e.png" height="600"/>

Reply:
@Undermaken you're very welcome :)

I removed the digit style (and its background) on purpose because I felt it makes the keypad look more balanced and symmetric.

Plus that char's being wider than the digits messes up the layout: on Android (and on iOS it looks like ![ios](https://user-images.githubusercontent.com/788293/81062992-4da51a00-8ed7-11ea-9ed1-db380188284a.png)) it's not centered in the circle and it doesn't even fully fit because it's a nudge too big.

WDYT about using its own style with no background circle and a slightly bigger font size (but smaller than the digits')?

Reply:
@bfabio 
> WDYT about using its own style with no background circle and a slightly bigger font size (but smaller than the digits')?
I'm pretty sure this could be the best solution